typescript/aleo-sdk/scripts/run-e2e-test.sh

```diff
@@ -24,7 +24,7 @@ export ALEO_SKIP_PROOFS=true
 export ALEO_SKIP_SUFFIXES=false
 export ALEO_UPGRADE_AUTHORITY=""
 export ALEO_CONSENSUS_VERSION_HEIGHTS="0,1,2,3,4,5,6,7,8,9,10,11"
-export ALEO_ISM_MANAGER_SUFFIX="custom"
+export ALEO_ISM_MANAGER_SUFFIX=""
```

Comment:
Do we still need this if it is just an empty string?

Reply:
readded again, temporary removed it and forgot to add it back